tox.ini

```diff
@@ -18,3 +18,12 @@ commands =
     python test.py
     python setup.py sdist
     python setup.py bdist_wheel
+
```

Comment:
Let's do it on both py27 and py36, that way we will cover more.

Reply:
You think this is a good idea? I realized for local development when you do "tox" you get bunch of warnings because it will try to do codecov when you develop locally too. Another approach is to explicitly use the coverage in the .tox/py36/bin environment.

Reply:
> will try to do codecov when you develop locally too. 

This should be conditioned by an env variable.

Here is how you can pass env var: https://github.com/j0057/github-release/blob/master/tox.ini